jep-006-improved-identifiers.md

```diff
@@ -98,14 +98,14 @@ in certain environments.  For example, in python, this is not a problem:
 >>> jmespath_expression = "foo\nbar"
 ```
 
-Python will interpret the sequence `"\\n"` (`%x5C %x6E`) as the newline
+Python will interpret the sequence `"\n"` (`%x5C %x6E`) as the newline
 character `%x0A`.  However, consider Bash:
 
 ```
 $ foo --jmespath-expression "foo\nbar"
 ```
 
-In this situation, bash will not interpret the `"\\n"` (`%x5C %x6E`)
+In this situation, bash will not interpret the `"\n"` (`%x5C %x6E`)
```

Comment:
I want to limit this PR to formatting changes only, but the double quotes inside these backticks are misleading and should be removed in a followup.

Reply:
Thanks for pointing this out.